Tools/clinic/clinic.py

```diff
@@ -846,6 +846,10 @@ class CLanguage(Language):
         static PyObject *
         {c_basename}({self_type}{self_name}, PyObject *Py_UNUSED(ignored))
     """)
+    PARSER_PROTOTYPE_GETTER: Final[str] = normalize_snippet("""
+        static PyObject *
+        {c_basename}({self_type}{self_name}, void *context)
```

Comment:
With the current implementation, `context` is always unused. Perhaps we should open up for it to be used? (We can add that feature in a follow-up PR.)

```suggestion
        {c_basename}({self_type}{self_name}, void *Py_UNUSED(context))
```

Reply:
> Perhaps we should open up for it to be used?

I think that we should support it if needed.
Maybe we should define optional params for using context or not.
IIRC, in the CPython code base, most of them are not using context directly, I think that we can support it as a separate PR :)

Reply:
I cannot recall a seeing a single use of the `context` param in the CPython code base :) If we the need arises, we can add support for it in a future PR.